code/modules/mob/new_player/new_player.dm

```diff
@@ -169,13 +169,6 @@
 		ViewManifest()
 
 	if(href_list["SelectedJob"])
-	
-		//Prevents people rejoining as same character.
-		for (var/mob/living/carbon/human/C in mob_list)
-			var/char_name = client.prefs.real_name
-			if(char_name == C.real_name)
-				usr << "<span class='notice'>There is a character that already exists with the same name - <b>[C.real_name]</b>, please join with a different one, or use Quit the Round with the previous character.</span>" //VOREStation Edit
-				return

```

Comment:
comment this out instead and add //CITADEL CHANGE - allows people to respawn as the same character